test/randomActionAgent.py

```diff
@@ -0,0 +1,70 @@
+import random
+import numpy as np
+from pathlib import Path
+
+from animalai import AnimalAIEnvironment, arenas
+from animalai.actions import AAIActions
+from animalai.executable import find_executable
+from animalai.agents import RandomActionAgent
+
+
+def main():
+    """
+    Manual test to see if random action agent works.
+    """
+
+    env = AnimalAIEnvironment(
+        file_name=str(find_executable(Path("test/executable/"))),
+        arenas_configurations=arenas.GOOD_GOAL_RANDOM_POS,
+        base_port=5005 + random.randint(0, 1000),
+        useRayCasts=True,
+        rayMaxDegrees=30,
+        raysPerSide=3,
+        play=False,
+        inference=True,
+    )
+
+    randomAgent = RandomActionAgent(step_length_distribution=lambda: np.random.normal(
+        5, 1))  # a Rayleigh walker (sampling from normal distribution)
+
+    # by default should be AnimalAI?team=0
```

Comment:
What you mean by this?

Reply:
Which part? a `Rayleigh Walker` is a type of random walker in stochastic process theory, where angles and saccades are drawn from normal distributions. The default to `AnimalAI?team=0` is the standard behaviour for the AAI low-level API.